pkg/virtualKubelet/forge/meta.go

```diff
@@ -3,7 +3,11 @@ package forge
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 const (
+	// LiqoOutgoingKey is a label to set on all offloaded resources.
```

Comment:
Completely OT question: Where do we use LiqoIncomingKey and LiqoOutgoingKey? @aleoli @giuse2596

Reply:
As far as I understand LiqoOutgoingKey is used as label to mark offloaded pods (or ReplicaSet) and you can find its usage watching function CreatePod in pkg/virtualKubelet/provider/provider.go
LiqoIncomingKey I think is unused but I am not sure.